src/hooks/useStreamingList.js

```diff
@@ -37,7 +39,7 @@ const useStreamingList = () => {
     }
   }, []);
 
-  return { streaming, error };
+  return { streaming, setStreaming };
```

Comment:
```suggestion
  return { streaming, setStreaming, error };
```
As prev. comment, we are not showing errors right now so we could:
1. Remove all `catch => setError` (I don't think this is a good one)
2. Return the `error` as we are currently doing in all files (even though we are not using them now)